client-py/SessionExample.py

```diff
@@ -178,18 +185,21 @@
 
 # insert one numpy tablet into the database.
 np_values_ = [
-    np.array([False, True, False, True], np.dtype('>?')),
-    np.array([10, 100, 100, 0], np.dtype('>i4')),
-    np.array([11, 11111, 1, 0], np.dtype('>i8')),
-    np.array([1.1, 1.25, 188.1, 0], np.dtype('>f4')),
-    np.array([10011.1, 101.0, 688.25, 6.25], np.dtype('>f8')),
+    np.array([False, True, False, True], np.dtype(">?")),
+    np.array([10, 100, 100, 0], np.dtype(">i4")),
+    np.array([11, 11111, 1, 0], np.dtype(">i8")),
+    np.array([1.1, 1.25, 188.1, 0], np.dtype(">f4")),
+    np.array([10011.1, 101.0, 688.25, 6.25], np.dtype(">f8")),
     np.array(["test01", "test02", "test03", "test04"]),
 ]
-np_timestamps_ = np.array([1, 2, 3, 4], np.dtype('>i8'))
+np_timestamps_ = np.array([4, 3, 2, 1], np.dtype(">i8"))
```

Comment:
Hi, how about adding a new example of inserting unsorted tablet instead of changing the existing example?